docs/quickstart_guide.md

```diff
@@ -258,11 +258,13 @@ This section is only for synchronous (i.e., Knative Serving) functions. Please r
 
 1. Run the invoker client:
     ```bash
-    go run examples/invoker/client.go
+    pushd ./examples/invoker && go build && popd && ./examples/invoker/invoker
     ```
 
     > **Note:**
     >
+    > The aforementioned command can only be run on the same node where the deployer command was invoked. In order to run the invoker on other nodes, the generated `endpoints.json` file has to be copied manually to the root directory of vHive.
```

Comment:
@cvetkovic can you instead say: copy the file produced by the deployer and run the invoker with the XX argument to specify the path and the name of that file

Reply:
Done. No need to specify any arguments if the JSON file is copied to vHive root.